packages/account-utils/src/PasswordPolicyClass.ts

```diff
@@ -1,8 +1,36 @@
-import { Random } from '@rocket.chat/random';
 import generator from 'generate-password';
-import { Meteor } from 'meteor/meteor';
+
+import { PasswordPolicyError } from './PasswordPolicyError';
 
 class PasswordPolicy {
+	regex: {
```

Comment:
I think it should be better to encapsulate all the properties as `private`. Wdyt?

Reply:
Hmm, I think those props can't be private, since we are updating them with the `settings.watch()` method on the backend... and the frontend uses the `useSetting()` hook to keep it synced with the backend instance.